core/src/main/resources/static/js/requestAvroSchemaUpload.js

```diff
@@ -144,6 +144,15 @@ app.controller("requestSchemaCtrl", function($scope, $http, $location, $window)
                         return;
                     }
 
+                    if($scope.addSchema.forceRegister === true) {
+                    var forceRegisterString = " Force register for schema selected. This overrides standard schema compatibility.";
+                        if($scope.addSchema.remarks == null) {
+                        $scope.addSchema.remarks = forceRegisterString
```

Comment:
```suggestion
                        $scope.addSchema.remarks = forceRegisterString;
```
not sure if this is required, but better